### PR TITLE
Document rank model

### PR DESCRIPTION
**Pull request by patrikgrenfeldt**
Added a short description of rank model version under "Misc" i Cases document

### CODE REVIEW

**Comment by patrikgrenfeldt** [comment]
run 'invoke docs' after merge

File: docs/admin-guide/loading-variants.md [hunk 1]
```diff
@@ -5,4 +5,8 @@ This document will describe the process of variant loading, how it is done and w
 ## Rank Score
 
 In Scout rank score is a central theme. Rank Scores is a estimation on how potentially dangerous a variant is, similar to the CADD score with the intention of work with all types of variation. 
-The uploading of variants is based on a rank score treshold, this is to avoid to clog the database with millions of variants that we at the moment have a hard time to say anything about.
\ No newline at end of file
+The uploading of variants is based on a rank score threshold, this is to avoid to clog the database with millions of variants that we at the moment have a hard time to say anything about.
+
+The rank score is calculated by the tool [GENMOD][genmod]
+
+[genmod]: http://exac.broadinstitute.org/faq
```
Comment (moonso) [inline docs/admin-guide/loading-variants.md]: Correct link should be https://github.com/moonso/genmod.

File: docs/admin-guide/loading-variants.md [hunk 2]
```diff
@@ -5,4 +5,8 @@ This document will describe the process of variant loading, how it is done and w
 ## Rank Score
 
 In Scout rank score is a central theme. Rank Scores is a estimation on how potentially dangerous a variant is, similar to the CADD score with the intention of work with all types of variation. 
-The uploading of variants is based on a rank score treshold, this is to avoid to clog the database with millions of variants that we at the moment have a hard time to say anything about.
\ No newline at end of file
+The uploading of variants is based on a rank score threshold, this is to avoid to clog the database with millions of variants that we at the moment have a hard time to say anything about.
+
+The rank score is calculated by the tool [GENMOD][genmod]
```
Comment (moonso) [inline docs/admin-guide/loading-variants.md]: Could we rephrase this to something more descriptive like:

"The rank score I a summary of the annotations of a variant. Rank scores are calculated and annotated by [GENMOD][genmod]"
